maven/bnd-export-plugin/src/main/java/aQute/bnd/build/StandaloneRun.java

```diff
@@ -0,0 +1,25 @@
+package aQute.bnd.build;
```

Comment:
Well, you only need to set the runbundles if you resolve during the build :-) I assume the runbundles could also be set by setting the -runbundles property in the Processor based upon a resolve run.